### ISSUE TITLE
2.2.4 only works with one vinyl

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#265]
Mixxx 2.2.3 on Ubuntu 20.04 LTS was working well with two turntables, NI Audio 8 DJ and the NI vinyls. After upgrading to 2.2.4 on the same OS, only vinyl control for one track works. Although the second turntable sends a valid timecode signal (checked in settings), the loaded track doesn't play (with vinyl control enabled for both tracks).

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2020-06-06T00:21:30Z
 * owner: resu1234
 * the launchpad url was https://bugs.launchpad.net/bugs/1882315

**Comment by Holzhaus** [comment]
(by daschuer)
Ups, that's a critical issue. 
Is the issue related to a specific deck?
Does it work if you configure vinyl control for deck 3 and 4?

Please start Mixxx try to vinyl control both decks and close Mixxx.
Tha post the ~/.mixxx/mixxx.log file

**Comment by Holzhaus** [comment]
(by sggwsoxzplkhphlsob)
I am happy to say that this can be closed!
Vinyl control worked for decks 3 and 4 - only deck 2 was not working. I decided to give you a fresh log file, so first I deleted the .mixxx directory. That solved the issue! I've configured vinyl control the same way as before and now deck 2 works again (deck 1 worked all the time).

Thanks for your prompt response and for the great work!
